SSA/Projects/InstCombine/Base.lean

```diff
@@ -127,7 +158,7 @@ namespace MOp
 @[match_pattern] def ashr   (w : Width φ) : MOp φ := .binary w .ashr
 @[match_pattern] def urem   (w : Width φ) : MOp φ := .binary w .urem
 @[match_pattern] def srem   (w : Width φ) : MOp φ := .binary w .srem
-@[match_pattern] def add    (w : Width φ) : MOp φ := .binary w .add
+@[match_pattern] def add    (w : Width φ) (nsw : Bool := false) (nuw : Bool := false) : MOp φ := .binary w (.add  ⟨ nsw , nuw ⟩ )
```

Comment:
Why does `BinaryOp.add` take its flags as a pair of Bools, but `MOp` takes them as two separate arguments?